doc/ale.txt

```diff
@@ -134,6 +134,9 @@ CONTENTS                                                         *ale-contents*
       phpstan.............................|ale-php-phpstan|
     pod...................................|ale-pod-options|
       write-good..........................|ale-pod-write-good|
+    proto.................................|ale-proto-options|
+      protoc..............................|ale-proto-protoc|
```

Comment:
Remove this entry and the documentation entry. There's no such linter with the name `protoc`.